terraform/modules/rds/password.tf

```diff
@@ -1,4 +1,4 @@
-resource "random_password" "password" {
+resource "random_password" "this" {
```

Comment:
I think the general thing on `this` is both "will there only be one" and "is it obvious what this is referring to from knowing what module it's in" - for this I could go either way on the latter question (since something like a random password you always want to know for what), but I do expect there will only be one. 🤷‍♀️ I'm happy with whatever here.